kolibri/content/utils/transfer.py

```diff
@@ -130,8 +129,14 @@ def close(self):
 class FileDownload(Transfer):
 
     def __init__(self, *args, **kwargs):
-        # Set block size to None, to always get the largest chunk available from the socket
-        kwargs["block_size"] = None
```

Comment:
@aronasorman Deleting this (so that it does 2MB blocks) actually sped things up slightly, and also gave more granular progress info.

Reply:
Cool! Hopefully it didn't break the progress bar.

Reply:
Break it? It fixed it! :D 

(For big files, turns it back into a progress bar instead of basically a boolean status indicator :P )